_docs/3-mi-v-soft-cpu.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: splash
+title: "Mi-V Soft CPUs"
+permalink: /docs/mi-v-soft-cpu/
+hidden: true
+header:
+  overlay_color: "#5e616c"
+  overlay_image: /assets/images/201106-CORP-BNR-Design-Centers-FPGAs-and-plds-Banner-2880x280.jpg
+excerpt: "Welcome to the Mi-V Soft CPU Page."
+last_modified_at: 2018-01-10T11:22:01-05:00
+---
+### Mi-V Soft CPUs
+Mi-V Ecosystem supports following RV32I base instruction set processors with optional extensions.
+* MIV_RV32
+* MIV_RV32IMA_L1_AHB * 
+* MIV_RV32IMA_L1_AXI * 
+* MIV_RV32IMAF_L1_AHB *
+
+\*Legacy cores. MIV_RV32 recommended for new designs
```

Comment:
Can this be aligned with the list items? i.e. align * with M in MIV_

Add a full stop.

Reply:
I can try this now. And will add the full stop.